core/proto/keytransparency_v1_types/keytransparency_v1_types.proto

```diff
@@ -96,14 +96,17 @@ message SignedKV {
   map<string, sigpb.DigitallySigned> signatures = 2;
 }
 
-// Mutation contains the actual mutation and the inclusion proof of the
-// corresponding leaf.
-message Mutation {
-  // update contains the actual mutation information.
-  SignedKV update = 1;
-  // proof contains a leaf and an inclusion proof in the map of the previous
-  // epoch. This is used by Storage-less monitors.
-  trillian.MapLeafInclusion proof = 2;
+// MutationProof contains the information nessesary to compute the new leaf value.
+// It contains the a) old leaf value with it's inclusion proof and b) the mutation.
+// The new leaf value is computed via:
+//       Mutate(leaf_value, mutation)
+message MutationProof {
+  // mutation contains the information needed to modify the old leaf.
+  // The format of a mutation is specific to the particular Mutate function being used.
+  // TODO: figure out if we want to keep this generic and embed proto.Any here instead.
```

Comment:
TODO(gdbelvin):

Reply:
Can you think of a situation where SignedKV couldn't fully express the mutation?

Reply:
Some applications might not want signatures in their leaf nodes. In general, it should be up to the application to decide what the format of their leaf nodes should look like.  At the moment though, we only have KT as the application.